Bottom: 476553bc1bb8f081e0227040bc8d65ad39afa5d0
Top:    9388f3c123b2256273500bebbb9028968387e1ed
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-14 12:31:04 -0700

Refresh of eliminate-virtual

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 3ee8bc607d7..fd63c53a451 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -507,7 +507,7 @@ class PythonList : public TypedPythonObject<PythonList> {
 public:
   using TypedPythonObject::TypedPythonObject;
 
-  PythonDictionary() : TypedPythonObject() {} // MSVC requires this for some reason
+  PythonList() : TypedPythonObject() {} // MSVC requires this for some reason
 
   explicit PythonList(PyInitialValue value);
   explicit PythonList(int list_size);
